tools/mpy-tool.py

```diff
@@ -1795,6 +1844,14 @@ def main(args=None):
         default=16,
         help="mpz digit size used by target (default 16)",
     )
+    cmd_parser.add_argument(
```

Comment:
I suggest moving this up to just after the `--merge` option, so they appear together in the help output (the action commands will then come before the tweaking options).

Reply:
Makes sense, thanks!  This will be addressed in the next PR iteration.